apps/external-devices/device-descriptors/Polhemus_Libert_Latus_tracker.json

```diff
@@ -0,0 +1,13 @@
+{
+    "deviceVendor": "Polhemus",
+    "deviceName": "Liberty Latus tracker",
```

Comment:
@rpavlik Turns out LIBERTY should be in all caps, too (per their website and docs).